packages/hooks/src/useCollectionQuery/useCollectionQuery.ts

```diff
@@ -256,9 +256,11 @@ function fetchMoreUpdateQueryHandler<TQuery>(
     );
   }
 
+  const totalCount = outputCollection.totalCount != undefined ? { totalCount: nextCollection.totalCount } : {};
```

Comment:
Won't it make sense to return `undefined` or `0` instead of empty object?
This way it will make it easier to work on the front end if we need to build some logic around `totalCount`, because `0` and `undefined` are both falsy values.

Reply:
The returned object needs to match what was passed in, in this case there is no `totalCount` so we shouldn't set it.